src/components/InputRange.vue

```diff
@@ -28,19 +28,32 @@ const Input = require('gui/inputs/input');
 export default {
   mixins: [Input],
   data() {
-    const options = this.state.input.options.values[0];
-    const min = 1*options.min;
-    const max = 1*options.max;
-    const step = 1*options.Step;
+    const {min, max, Step} = this.state.input.options.values[0];
     return {
-      max,
-      min,
-      step: step
+      min: 1*min,
+      max: 1*max,
+      step: 1*Step
     }
   },
```

Comment:
This portion of code may be obscure, add a comment above explaining why we need to multiply by 1